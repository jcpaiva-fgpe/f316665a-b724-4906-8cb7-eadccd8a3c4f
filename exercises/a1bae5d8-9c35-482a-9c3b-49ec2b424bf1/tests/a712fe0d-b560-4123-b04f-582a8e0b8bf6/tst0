-1 0
10
10
6 100
-1 10
3 2
-3 2
7 6
10 -1
7 25
-1 0
8 5
25 -1000
