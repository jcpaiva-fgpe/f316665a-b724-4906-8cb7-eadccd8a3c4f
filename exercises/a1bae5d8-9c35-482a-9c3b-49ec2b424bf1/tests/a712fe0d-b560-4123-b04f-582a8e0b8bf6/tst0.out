(6,100): no exterior
(-1,10): na fronteira
(3,2): no interior
(-3,2): no interior
(7,6): na fronteira
(10,-1): no exterior
(7,25): no exterior
(-1,0): no interior
(8,5): no exterior
(25,-1000): no exterior
